src/lib/CollectStore.js

```diff
@@ -145,7 +145,7 @@ export default class CollectStore {
               arguments: '0 0 0 * * *',
               worker: 'konnector',
               worker_arguments: {
-                konnector: connection.konnector.attributes.slug,
+                konnector: connection.konnector.slug,
```

Comment:
I tested it also for the first run and it worked. I also had this idea but I don't want to add more uncertainty in this code. I will add an explicit console.warn about this if connection.konnector.slug is not present. @gregorylegarec Are you ok with that?